pyproject.toml

```
@@ -31,6 +31,7 @@ gen3authz = "^0.4.0"
 gen3cirrus = "^1.3.0"
 gen3config = "^0.1.7"
 gen3users = "^0.6.0"
+idna = "^2.10"
```

```suggestion
idna = "^2.10" # https://github.com/python-poetry/poetry/issues/3555
```
i think pyproject.toml accepts comments